test/system/150-login.bats

```diff
@@ -398,6 +398,84 @@ EOF
     assert "$output" =~ "--retry-delay .*pull failures \(default \"5s\"\)"
 }
 
+@test "push and pull zstd chunked image" {
+    iid=localhost:${PODMAN_LOGIN_REGISTRY_PORT}/$(random_string 10 | tr A-Z a-z)
```

Comment:
Could you rename this please? To me, `iid` refers to a sha.

And, there's a new helper in town, `safename`. The preferred way to do that now would be
```bash
    .../img-$(safename)
```